docs/cudf/source/basics/basics.rst

```diff
@@ -15,32 +15,61 @@ The following table lists all of cudf types. For methods requiring dtype argumen
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Strings                |                              | `str <https://docs.python.org/3/library/stdtypes.html#str>`_                        | ``'string'``, ``'object'``                  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Datetime               |                              | np.datetime64_                                                                      | ``'datetime64[s]'``, ``'datetime64[ms]'``,  |
+    |                        |                              |                                                                                     | ``'datetime64[us]'``, ``'datetime64[ns]'``  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Timedelta              |                              | np.timedelta64_                                                                     | ``'timedelta64[s]'``, ``'timedelta64[ms]'``,|
+    | (duration type)        |                              |                                                                                     | ``'timedelta64[us]'``, ``'timedelta64[ns]'``|
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Categorical            | CategoricalDtype             | (none)                                                                              | ``'category'``                              |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Boolean                |                              | np.bool_                                                                            | ``'bool'``                                  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Decimal                | Decimal64Dtype               | (none)                                                                              | (none)                                      |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
```

Comment:
Are there any changes here, or is the diff picking up something non-material related to formatting?

Reply:
Yes, I had to reformat this chart so that it would auto fit upon rendering